src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  bookList: [],
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducer: {
+    checkStatus: (state) => ({
+      ...state,
+      categories: 'Under construction',
+    }),
```

Comment:
- [ ] Please update the categories slice for the check status action it should return the string "Under Construction"  and In case of unknown action, it should return the current state as requested in the project requirements.
  - You can consider the below code.
```javascript
 checkStatus: (state, action) => {
      state.items =
        action.payload === "Under construction"
          ? "Under construction"
          : state.items
    },
```

Reply:
@thecodechaser I didn't see it, thank you for the explanation :)